.github/workflows/main.yml

```diff
@@ -0,0 +1,29 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the "master" branch
+  push:
```

Comment:
Doesn't this need to be `push: master` or
```
push:
  - master
```

does push with omitted branch, take the default branch?